### ISSUE TITLE
Temporary database window and pgn game window do not remember size preferences

### ISSUE THREAD

**Issue by futurelauncher** [lukasmonk/lucaschessR#73]
Hi,
Lucaschess's main window always remembers minimization/maximization preference for the user, but this never happens with temporary database window and pgn game window. Always user has to maximize it for each and every single game pasted there.

**Comment by lukasmonk** [comment]
It is saved the width and height not the maximize state.

**Comment by futurelauncher** [comment]
This time instead of maximize, I dragged the window and enlarged it to the full size, but still after closing it and reopening it, it opens in a default size.
Database window is not a concern here, as it opens almost fully (95%), but specifically pgn game window opens 3/4 of monitor screen. The logic is, if you are going to study a game, you would prefer full screen, and then for each new game you have to enlarge (maximize) it. It would be great if it remembers user choice.

**Comment by lukasmonk** [comment]
The reading of pgn file is done by creating a temporary database.
All databases use the same window, including the temporary one, size are shared by all.
In all my tests the windows are saved and retrieved fine.
Obviously I am not understanding something.

**Comment by futurelauncher** [comment]
Let us go step by step.
1) I open software. First window is this -
![image](https://user-images.githubusercontent.com/60097814/150680153-c5f938ea-e230-43a9-9c5e-a7f7f51d38e5.png)
2) I paste a single game with Alt+1 shortcut. Temporary database window opens as this -
![image](https://user-images.githubusercontent.com/60097814/150680400-23d2df76-888b-4433-94bc-19d85d986782.png)
3) Now I press enter key and open a game like this -
![image](https://user-images.githubusercontent.com/60097814/150680479-981a7320-c33e-4040-a810-760784c87829.png)
Now look at the size of this game window. It is always 3/4 of monitor screen. To enlarge it, I click maximize icon on top right. There should be some way to have this window in full size by default. Of course this is a small thing, not a big problem, but it can simply save a click of any user. Hopefully I am pretty much clear this time. Let me know if you didn't understand any part of this comment.

**Comment by lukasmonk** [comment]
I was in the previous step.
You can change the size of the board by clicking on the tool on the left upwards.

**Comment by futurelauncher** [comment]
Sorry, I cannot locate that tool. Where is it exactly?
![18527a5b-6945-4b1e-a685-040895e82fd7](https://user-images.githubusercontent.com/60097814/150683677-e1f3380a-e4be-4c48-94eb-f3a02e074625.jpg)

**Comment by lukasmonk** [comment]
You have to put it in a normal state, not maximised.

**Comment by futurelauncher** [comment]
It is in a normal state. There are only 6 visible options viz. colors, pieces, default, director, flip the board & active keys. No option to change the size of the board.

**Comment by lukasmonk** [comment]
The only reason for not showing the resizing is when it is maximised.

**Comment by futurelauncher** [comment]
This is the screenshot of exactly what I am seeing. This is the screenshot of default window after opening the game of temporary database.

![image](https://user-images.githubusercontent.com/60097814/150686786-6a788208-7b02-4f0a-b560-91a7f9728211.png)

**Comment by lukasmonk** [comment]
There are two files that control this appearance: 
```
\UserData\__Config__\confvid.pkd
\UserData\__Config__\confBoards.pk
```
If you send them to me I might be able to replicate the problem.

**Comment by futurelauncher** [comment]
Sent you at lucasmonk@gmail.com.

**Comment by lukasmonk** [comment]
lukasmonk must be

**Comment by futurelauncher** [comment]
Ops, sorry...corrected and sent.

**Comment by lukasmonk** [comment]
You can change the size with CTRL and the mouse wheel.
If the main board is maximised, then the change is not possible through the menu, I have to check this.

**Comment by futurelauncher** [comment]
Yes, you caught a bug here. Anyways, even though you change the size of the board, it is increased in the bottom portion (half issue resolved), but not towards right side. 
This is what I get now -
![image](https://user-images.githubusercontent.com/60097814/150889348-38d77f89-c994-4fc1-9d19-3e093ab6362c.png)

And this is what I expect Lucaschess to remember at the time of closing the window or program.
![image](https://user-images.githubusercontent.com/60097814/150889552-041c82d8-2f32-4941-b62d-9e0fd65672b2.png)
But unfortunately it doesn't remember this while reopening the program or a game and goes back to first image size. Here I am referring size of the whole game window, not the board. I hope, I am pretty much clear now.

**Comment by lukasmonk** [comment]
Fixed in the next update (version R2).

**Comment by futurelauncher** [comment]
This issue is not resolved yet in R2.

**Comment by lukasmonk** [comment]
I think in the next update, this time it will be solved.
